cyvcf2/cli.py

```diff
@@ -0,0 +1,159 @@
+import logging
+from datetime import datetime
+
+import coloredlogs
+import click
+
+from . import VCF
+
+from . import __version__
+
+log = logging.getLogger(__name__)
+LOG_LEVELS = ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
+
+
+def print_header(vcf_obj, include, exclude, silent=False):
+    """Print the header of a vcf obj
+    
+    Parameters
+    ----------
+    vcf_obj: cyvcf2.VCF
+    include: tuple
+        set of strings with info fields that should be included
+    exclude: tuple
+        set of strings with info fields that should be excluded
+    silent: bool
+        If stuff should be printed or not
+    """
+    for header_line in vcf_obj.raw_header.split('\n'):
+        if len(header_line)>0:
+            print_line = True
+            if include:
+                if 'INFO' in header_line:
+                    print_line = False
+                    for inclusion in include:
+                        if inclusion in header_line:
```

Comment:
I missed this first time. A few minor things, but first, this could be simplified to:
```
print_line = any(inc in header_line for inc in include)
```
and same below.
Second, the match should be more careful, especially for exclude. E.g. if you exclude "AF" and then there's a field in the header called "gnomad_AF", then the gnomad_AF field would be excluded. so, 
maybe change to something like:

```
print_line = any("ID=" + inc + "," in header_line for inc in include)
```

my other nit is to avoid nesting (from python koan: "flat is better than nested") by replacing:
```
if 'INFO' in header_line:
    <indented code block>
```
with
```
if not 'INFO' in header_line:
    continue
<code block>
```

similarly, just put: `if silent: return` at the start of the function and then remove those checks from the body.